app/serializers/rest/application_serializer.rb

```diff
@@ -2,7 +2,10 @@
 
 class REST::ApplicationSerializer < ActiveModel::Serializer
   attributes :id, :name, :website, :scopes, :redirect_uri,
-             :client_id, :client_secret, :vapid_key
+             :client_id, :client_secret
+
+  # TODO: Deprecated in 4.3.0, needs to be removed in 5.0.0
```

Comment:
```suggestion
  # NOTE: Deprecated in 4.3.0, needs to be removed in 5.0.0
```

@renchap I don't know which is better, up to you.